exclude_files.txt

```diff
@@ -0,0 +1 @@
+*.py
```

Comment:
this excludes file should be deleted. even though this a revert, the build job itself has changed to globally exclude .pyc files, so excluding .py will result in no code files being included in the tarball